index.js

```diff
@@ -23,17 +23,26 @@ app.use(bodyParser.json())
 app.use(msCorrelation.correlationMiddleware)
 
 const tournamentDataRouter = require('./routers/tournamentDataRouter')
-const matchTeamRouter = require('./routers/matchTeamRouter');
+const matchTeamRouter = require('./routers/matchTeamRouter')
+
+app.use((req, res, next) => {
```

Comment:
For people unfermilliar with the background workings of `cors` requests, this would make no sense. Also there are a couple of headers missing. You can just use the [cors](https://www.npmjs.com/package/cors) package which is meant for this.